modules/asteriobidAnalyticsAdapter.md

```diff
@@ -0,0 +1,8 @@
+# Overview
+
+Module Name: AsterioBid Analytics Adapter
+Module Type: Analytics Adapter
+Maintainer: admin@asteriobid.com
+
+# Description
+Analytics adapter for <a href="https://asteriobid.com/">AsterioBid</a>. Contact admin@asteriobid.com for information.
```

Comment:
Could you include information about your config options?

Reply:
I added the description of test and advanced configuration parameters